misc/snapshotter/Dockerfile

```diff
@@ -4,26 +4,32 @@ ARG NYDUS_VER=v2.1.5
 
 RUN apk add --no-cache curl
 RUN apk add --no-cache --upgrade grep
-RUN wget https://github.com/dragonflyoss/nydus/releases/download/$NYDUS_VER/nydus-static-$NYDUS_VER-linux-amd64.tgz && \
+RUN curl -OL https://github.com/dragonflyoss/nydus/releases/download/$NYDUS_VER/nydus-static-$NYDUS_VER-linux-amd64.tgz && \
     echo $NYDUS_VER > /.nydus_version && \
     tar xzf nydus-static-$NYDUS_VER-linux-amd64.tgz && \
-    rm nydus-static-$NYDUS_VER-linux-amd64.tgz
-RUN mv nydus-static/* /
+    rm nydus-static-$NYDUS_VER-linux-amd64.tgz && \
+    mv nydus-static/* / && \
+    curl -OL https://github.com/liubin/toml-cli/releases/download/v0.0.7/toml-cli-v0.0.7-linux-amd64.tgz && \
+    tar xzf toml-cli-v0.0.7-linux-amd64.tgz && \
+    mv toml-cli/toml / && \
+    rm toml-cli-v0.0.7-linux-amd64.tgz
 
 FROM alpine:3.17.0
 
 WORKDIR /root/
 
-RUN apk add --no-cache libc6-compat
+RUN apk add --no-cache libc6-compat bash jq
```

Comment:
Hi @ChengyuZhu6 ! I'm not sure if you are aware of but we have found an issue while using toml-cli on kata containers that was corrupting the containerd's configuration file. @fidencio gave some details in https://github.com/kata-containers/kata-containers/pull/8679#issuecomment-1864556280 but it turns out you must ensure jq > 1.7 is installed. Here is how it was solved on kata side: https://github.com/kata-containers/kata-containers/pull/8709/commits/41320c586ef80c5d15467dbf31a059be4bab7065

Reply:
Hey @wainersm. Thanks for your review. Here we use `toml-cli` , not `tomlq`.

Reply:
oh, I was wondering with the binary is named toml rather than tomlq and that explains :)